.github/workflows/security-secrets.yml

```
@@ -75,3 +82,4 @@ jobs:
           GITHUB_DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
           GITGUARDIAN_API_KEY: ${{ secrets.GITGUARDIAN_API_KEY }}
           GITHUB_TOKEN: ${{ secrets.BOT_TOKEN || github.token }}
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)